src/redux/reducers/userProfileReducer/userProfileReducer.test.js

```diff
@@ -0,0 +1,58 @@
+import userProfileReducer from './userProfileReducer';
+import {
+    PROFILE_ERROR,
+    PROFILE_DATA,
+    PROFILE_PATCH_START,
+    PROFILE_PATCH_SUCCESS,
+    PROFILE_PATCH_ERROR,
+    PROFILE_DATA_SUCCESS,
+  } from '../../actions/userProfileActions/types';
+
+describe('userProfileReducer tests', () => {
+  it('it should have an initial state', () => {
+    expect(userProfileReducer(userProfileReducer.initialState, {})).toEqual({
+        profileData: {},
+        error: {},
+        profileUpdate: false,
+        isLoading: false,
+        Followers: 0,
+        Following: 0,
+});
+  });
+  it('it should show isloading on profile submit', () => {
```

Comment:
isLoading should be set to true instead on submit. You can check this test again.

Reply:
@jomasim  
Thank you for the note.
The test is okay, but not the narration which I have edited. Thank you.